### DISCUSSION TITLE
Ability to Preserve Source IPs with Proxy Stream

### DISCUSSION THREAD

**Discussion by bajascripts** [NginxProxyManager/nginx-proxy-manager#2641] [Ideas]
Hello, I have a tiny suggestion.
Currently, when using Proxy Stream, if you use it for, let's say, Minecraft, it will cover up the source IP for the player and set it to the local ip NPM is running on. Instead, it would be nice to have an option to preserve the user's source IP.

**Comment by neocult-de** [comment]
What worked for me is letting the docker container bind directly to the host port, so it doesn´t use the docker-proxy to be forwarded and thus changing the source IP. Add 0.0.0.0 to your ports to bind on all Interfaces (Internal docker networks your NPM container is attached to AND the host port directly)

To make that happen adopt your docker run command ` -p  0.0.0.0:25565:25565`   (Adding 0.0.0.0 to your port forwarding for the npm-docker image)

If you use docker-compose 
`
ports:
        0.0.0.0:25565:25565
`   

Worked for me on Port 80/443 to have the SourceIP
